tests/components/pushbullet/test_notify.py

```diff
@@ -1,301 +1,296 @@
-        assert len(mock.request_history) == 2
+    data = {
+        "title": "Test Title",
+        "message": "Test Message",
+        "target": ["device/DESKTOP", "device/My iPhone"],
+    }
+    await hass.services.async_call(notify.DOMAIN, "test", data)
+    await hass.async_block_till_done()
+    assert requests_mock.called
+    assert requests_mock.call_count == 2
+    assert len(requests_mock.request_history) == 2
 
-        expected_body = {
-            "body": "Test Message",
-            "device_iden": "identity1",
-            "title": "Test Title",
-            "type": "note",
-        }
-        assert mock.request_history[0].json() == expected_body
-        expected_body = {
-            "body": "Test Message",
-            "device_iden": "identity2",
-            "title": "Test Title",
-            "type": "note",
-        }
-        assert mock.request_history[1].json() == expected_body
+    expected_body = {
+        "body": "Test Message",
+        "device_iden": "identity1",
+        "title": "Test Title",
+        "type": "note",
+    }
+    assert requests_mock.request_history[0].json() == expected_body
+    expected_body = {
+        "body": "Test Message",
+        "device_iden": "identity2",
+        "title": "Test Title",
+        "type": "note",
+    }
+    assert requests_mock.request_history[1].json() == expected_body
 
-    @requests_mock.Mocker()
-    @patch.object(
-        PushBullet,
-        "_get_data",
-        return_value=json.loads(load_fixture("pushbullet_devices.json")),
-    )
-    def test_pushbullet_push_email(self, mock, mock__get_data):
-        """Test pushbullet push to default target."""
-        config = {
-            notify.DOMAIN: {
-                "name": "test",
-                "platform": "pushbullet",
-                "api_key": "MYFAKEKEY",
-            }
-        }
-        with assert_setup_component(1) as handle_config:
-            assert setup_component(self.hass, notify.DOMAIN, config)
-        assert handle_config[notify.DOMAIN]
-        mock.register_uri(
-            requests_mock.POST,
-            "https://api.pushbullet.com/v2/pushes",
-            status_code=200,
-            json={"mock_response": "Ok"},
-        )
-        data = {
-            "title": "Test Title",
-            "message": "Test Message",
-            "target": ["email/user@host.net"],
-        }
-        self.hass.services.call(notify.DOMAIN, "test", data)
-        self.hass.block_till_done()
-        assert mock.called
-        assert mock.call_count == 1
-        assert len(mock.request_history) == 1
 
-        expected_body = {
-            "body": "Test Message",
-            "email": "user@host.net",
-            "title": "Test Title",
-            "type": "note",
+@patch.object(
+    PushBullet,
+    "_get_data",
+    return_value=json.loads(load_fixture("pushbullet_devices.json")),
+)
+async def test_pushbullet_push_email(mock__get_data, hass, requests_mock):
```

Comment:
Instead of doing this on all the test functions, make it a fixture. 

Add to top of file:

```python
@pytest.fixture
def mock_pushbullet():
    """Mock pushbullet."""
    with patch.object(
        PushBullet, "_get_data",  
        return_value=json.loads(load_fixture("pushbullet_devices.json"))
    ):
        yield
```

And then for each test:

```suggestion
async def test_pushbullet_push_email(mock__get_data, hass, requests_mock, mock_pushbullet):
```